assets/css/_sass/website/_component-page.scss

```diff
@@ -51,13 +56,6 @@
     color: $grey900;
     line-height: $base-line-height;
   }
-  .hero-image {
```

Comment:
Are we taking this out because the images will now be standardized when they're made?

Reply:
yep!  Evan created a nice little thumbnail set for us, based on Ty's visual design.  We'll be using those for the family page images and the cyan header image.

Reply:
Just a heads up regarding the thumbnails. They are all being served from cloud files CDN atm. Once we get more content on the site we can go back and update the liquid logic and upload all the thumbnails to this repo. Just calling it out because it's discussed and is on my to do later list.

Reply:
👍